commit 1d42b8f293c55226d18d97127419157fa4e47b41
Author: Dan Williams <dcbw@redhat.com>
Date:   Thu May 10 22:53:14 2007 -0400

    [PATCH] libertas: remove 8021xauthalgs private ioctl
    
    Useless; it set an internal variable that was unused anyway.
    A supplicant handles all 802.1x authentication, not the driver.
    
    Signed-off-by: Dan Williams <dcbw@redhat.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/libertas/wext.c b/drivers/net/wireless/libertas/wext.c
index 4a52336bc0f6..81ee83cb001b 100644
--- a/drivers/net/wireless/libertas/wext.c
+++ b/drivers/net/wireless/libertas/wext.c
@@ -1087,12 +1087,6 @@ static const struct iw_priv_args wlan_private_args[] = {
 	 IW_PRIV_TYPE_NONE,
 	 "authalgs",
 	 },
-	{
-	 WLANSET8021XAUTHALG,
-	 IW_PRIV_TYPE_INT | IW_PRIV_SIZE_FIXED | 1,
-	 IW_PRIV_TYPE_NONE,
-	 "8021xauthalgs",
-	 },
 	{
 	 WLANSETENCRYPTIONMODE,
 	 IW_PRIV_TYPE_INT | IW_PRIV_SIZE_FIXED | 1,